src/psyclone/psyir/transformations/acc_kernels_trans.py

```diff
@@ -133,8 +141,13 @@ def validate(self, nodes, options=None):
             a routine that is not available on the accelerator.
         :raises TransformationError: if there are no Loops within the
             proposed region and options["disable_loop_check"] is not True.
+        :raises TransformationError: if any assignments in the region contain a
+            character type child and the allow_strings option is not set.
```

Comment:
the option is 'allow_string', I think maybe `'allow_string'` inside quotes or something here to make it clear. Alternatively match the previous Error docstring for this and just use `options["allow_string"] is not True.

Reply:
Good idea, done.